packages/framework/src/Migrations/Version20250303130234.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20250303130234 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $this->sql('ALTER TABLE promo_code_limit DROP CONSTRAINT promo_code_limit_pkey');
```

Comment:
do not we want that constraint anymore? It is covered by that new primary key, right?

Reply:
Yes